runtime/README.md

```diff
@@ -187,6 +187,26 @@ Table: `STASH_MEASUREMENT` output arguments
 | result      | u32      | Result code. Little endian.
 | dpe\_result | u32      | Result code of DPE DeriveChild command. Little endian.
 
+### DISABLE\_ATTESTATION
+
+Disable attestation by erasing the CDI and DICE key. This command is intended
+for callers who update infrequently and cannot tolerate a changing DPE API surface, and is intended for situations where Caliptra firmware cannot be loaded
+and the SoC must proceed with boot.
+
+Upon receipt of this command, the current CDI is replaced with zeroes, and the
+associated DICE key is re-derived from the zeroed CDI.
```

Comment:
> the associated DICE key is re-derived from the zeroed CDI

I don't quite understand this. This seems to imply Caliptra firmware would still wield a signing key for the caller. Presumably we want to (1) at least refuse to give out a cert/CSR and (2) consider not signing stuff. Right?

Reply:
We could do that too. My thinking was that the low-level machinery all works the same so it may be easier to run testing flows in certain circumstances. Con is that it may be less obvious that something has gone wrong; certs will just be invalid. Could define some error state for DPE where it's told by the Caliptra boot stack that something has gone wrong and it should just fail to sign anything.

Reply:
Why will the certificate be wrong? It will still be signed by RT Alias, will still contain whatever measurements are made in the future. The key won't be stable but it'll still be valid to a verifier.

Reply:
If we re-derive the current DICE key (either LDevID, AliasFMC, or AliasRT depending on when `DISABLE_ATTESTATION` is invoked) based on a zeroed CDI, then when DPE in RT fw goes to sign a DPE leaf cert, some link in that cert chain will break because the valid keypair endorsed by one cert won't be the zeroed key used to sign the next cert.

Reply:
Oh I see. I thought you meant the current DPE CDI.

Is there a specific KeyVault slot we can refer to which holds the "current" CDI? I guess it's different for each layer. Maybe would be good to specify "The CDI for the current Caliptra boot layer"

Reply:
Each layer actually uses the [same slot](https://github.com/chipsalliance/caliptra-sw/blob/6581104f46041eea9796de047431faa6f0b7bb27/rom/dev/src/flow/cold_reset/mod.rs#L32) for its CDI.

Reply:
Oh cool. Maybe just change "current CDI" to "current Caliptra firmware CDI" for clarity?

Reply:
Changed to "Caliptra's current CDI", since it might not be a firmware CDI yet (if this is called before firmware download).